Beta v7.1.2

MichaIng opened the pull request:
### Beta v7.1.2
_(2021-04-28)_

#### Changes since v7.1.1
- DietPi-Software | RPi.GPIO: This software option has been renamed to "Python 3 RPi.GPIO" to make clear that it is a Python package. In our efforts to migrate all software options to Python 3, only the Python 3 package is installed from now on. To install it for Python 2, one needs to run the following command manually form console: "apt install python-rpi.gpio"

#### Fixes since v7.1.1
- DietPi-Config | Worked around an issue in RPi, where selecting a sound card ended with an error prompt in certain circumstances. Many thanks to @sturbs for reporting this issue: https://github.com/MichaIng/DietPi/issues/4306
- DietPi-Software | WebIOPi: Resolved an issue where the GPIO pins could not be toggled via web interface by moving to a newer fork of this project. This additionally enabled us to run it with Python 3 and enable it on RPi 3 and RPi 4 models. Many thanks to `@torwan` for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=8945
- DietPi-Software | Plex Media Server: Resolved an issue where secure remote access via app.plex.tv did not work if Unbound with DNS rebinding protection is installed. Many thanks to `@danmo117` for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=8896
- DietPi-Software | MATE: Resolved an issue where DietPi menu and desktop shortcuts could not be started, as the MATE terminal emulator is not compatible with the "start in console" flag of desktop shortcuts. xterm is now installed together with MATE, to replace the default terminal emulator. Many thanks to `@maya95` for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=8949

MichaIng committed:
v7.1

+ DietPi-Software | Unbound: Add root hints cron job path to comment in config file

MichaIng committed:
v7.1

+ DietPi-Set_hardware | Work around an issue where on RPi "alsactl -g store" can fail in circumstances that need to be investigated further
+ DietPi-Set_hardware | RPi OpenGL: Do not force a specific framebuffer size as e.g. the console frequency can still be chosen freely and depending on the VC4 mode, the framebuffer otherwise is not used.
+ DietPi-Set_hardware | Minor coding, especially further apply G_EXEC error handler where applicable

MichaIng committed:
v7.1

+ CHANGELOG | DietPi-Config: Worked around an issue in RPi, where selecting a sound card ended with an error prompt in certain circumstances
+ CHANGELOG | Set v7.1 release PR URL

MichaIng committed:
v7.1

+ RC up

MichaIng committed:
v7.1

+ DietPi-Globals | Default RC up
+ DietPi-Globals | Minor coding

MichaIng committed:
v7.1

+ DietPi-Create_MySQL_DB | Fix exit code and make read-only variables read-only

MichaIng committed:
v7.1

+ DietPi-LetsEncrypt | Lighttpd on Bullseye: Keep session tickets enabled, which is save since Lighttpd v1.4.56: https://github.com/MichaIng/DietPi/issues/4294#issuecomment-826802056

MichaIng committed:
v7.1

+ DietPi-LetsEncrypt | Typo

MichaIng committed:
v7.1

+ DietPi-DDNS | Fix Dyno response check

MichaIng committed:
v7.1

+ DietPi-Software | Minor renaming, description and code alignments around RPi GPIO-related install options

MichaIng committed:
v7.1

+ DietPi-Software | When doing npm installs, skip audits, as the produced warnings are meant for developers, not such much for end users, which might more be confused if something went wrong. Keep the "X packages are looking for funding\n  run `npm fund` for details" output, as this is fair towards the package developers.

MichaIng committed:
v7.1

+ DietPi-VPN | Use the single 1194 port only, so that the connection is predictable for users to configure firewalls accordingly. Future releases may add an option to switch between supported ports or even add multiple ports to use randomly (remote-random), if wanted.

MichaIng committed:
v7.1

+ DietPi-VPN | ProtonVPN configs now use port 1194 only, hence simplify the killswitch rules. Future releases may add an option to switch between supported ports or even add multiple ports to use randomly (remote-random), if wanted, but for now only the default OpenVPN port 1194 is supported.

nonnorm commented on .conf/dpv/protonvpn.template:
```diff
@@ -26,12 +26,7 @@
-remote $VPN_SERVER 5060
-remote $VPN_SERVER 4569
 
 remote-random
```
You might want to remove this line too.
**EDIT**: Ah, I just saw your other comment, nevermind.

MichaIng replied:
Jep, not 100% sure on this, had the same idea, but I was just thinking that NordVPN might have a reason and at least it is only a noop in case 😄.

MichaIng committed:
v7.1

+ CHANGELOG | Resolved an issue where the GPIO pins could not be toggled via web interface by moving to a newer fork of this project

MichaIng committed:
v7.1

+ DietPi-Set_software | The Raspberry Pi repository has now a Bullseye suite:

MichaIng committed:
v7.1

+ DietPi-PREP | Tiny

MichaIng committed:
v7.1

+ DietPi-Banner | Fix default selection for VMs vs non-VMs

MichaIng committed:
v7.1 (#4310)

+ DietPi-Software | WebIOPi: Use newer fork to enable RPi 3 and RPi 4 support and make it a Python 3 only install. Remove the Python package(s) on uninstall as well.
+ DietPi-Software | Make RPi.GPIO a Python 3 only install since all dependant software packages now support Python 3. Simplify and align the dependency declaration.
+ DietPi-Software | WebIOPi: Install native systemd unit to replace the SysV service
+ DietPi-Software | WebIOPi: Apply process shutdown fix until it has been applied to the fork we use
+ DietPi-Software | WebIOPi: Remove Google Analytics code from JavaScript, I can't believe it...

MichaIng committed:
v7.1

+ DietPi-Software | Plex Media Server: When Unbound is installed, add "plex.direct" to private domains to fix secure remote access via app.plex.tv: https://dietpi.com/phpbb/viewtopic.php?t=8896

MichaIng committed:
v7.1

+ CHANGELOG | Plex Media Server: Resolved an issue where secure remote access via app.plex.tv did not work if Unbound with DNS rebinding protection is installed

MichaIng committed:
v7.1

+ DietPi-Patches | Plex Media Server: Fix secure remote access via app.plex.tv did not work if Unbound with DNS rebinding protection is installed
+ DietPi-Patches | Fix No-IP install detection

MichaIng committed:
v7.1

+ DietPi-DDNS | Add Dynu to CLI usage and add info which options are required for which provider

MichaIng committed:
v7.1

+ DietPi-Software | MATE: Re-add xterm, as the mate-terminal is not compatible with desktop console shortcuts

MichaIng committed:
v7.1

+ CHANGELOG | MATE: Resolved an issue where DietPi menu and desktop shortcuts could not be started, as the MATE terminal emulator is not compatible with the "start in console" flag of desktop shortcuts. xterm is now installed together with MATE, to replace the default terminal emulator. Many thanks to @maya95 for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=8949
